cmd/registry-disk-v1alpha/entry-point.sh

```diff
@@ -54,6 +54,11 @@ fi
 echo "copied $IMAGE_PATH to $COPY_PATH.${IMAGE_EXTENSION}"
 
 touch /tmp/healthy
-while [ -f "${COPY_PATH}.${IMAGE_EXTENSION}" ]; do
-	sleep 5
-done
+if [ -n "$AS_ISCSI" ];
+then
+	bash expose-as-iscsi.sh "${COPY_PATH}.${IMAGE_EXTENSION}"
```

Comment:
There's no reason to copy the image to another location (assuming the image is already raw format) if iscsi is exposing it.